internal/tiltfile/tiltfile.go

```diff
@@ -351,25 +378,28 @@ func Load(ctx context.Context, filename string) (*Tiltfile, error) {
 	return tiltfile, nil
 }
 
-func (t Tiltfile) GetManifestConfigs(names ...string) ([]model.Manifest, error) {
+// GetManifestConfigsAndGlobalYaml executes the Tiltfile to create manifests for all resources and
+// a manifest representing the global yaml
+func (t Tiltfile) GetManifestConfigsAndGlobalYaml(names ...string) ([]model.Manifest, model.YAMLManifest, error) {
```

Comment:
nit: rename to `GetManifestsConfigsAndGlobalYAML` per [Golang's initialism policy](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).